src/components/CharacterPortrait.vue

```diff
@@ -46,17 +49,31 @@ export default defineComponent({
         : characterInfo.value?.metas.speakerName;
     });
 
+    const characterEngineName = computed(() => {
+      const activeAudioKey = store.getters.ACTIVE_AUDIO_KEY;
+      const audioItem = activeAudioKey
+        ? store.state.audioItems[activeAudioKey]
+        : undefined;
+      const engineId = audioItem?.engineId ?? store.state.engineIds[0];
+      const engineInfo = store.state.engineInfos[engineId];
+      return engineInfo?.name;
+    });
+
     const portraitPath = computed(() => characterInfo.value?.portraitPath);
 
     const isInitializingSpeaker = computed(() => {
       const activeAudioKey = store.getters.ACTIVE_AUDIO_KEY;
       return store.state.audioKeyInitializingSpeaker === activeAudioKey;
     });
 
+    const isEngineMultiple = store.state.engineIds.length > 1;
```

Comment:
めちゃめちゃ細かいのですが、Multipleは形容詞なので`isMultipleEngine`のが意図に合ってそうです

```suggestion
    const isMultipleEngine = store.state.engineIds.length > 1;
```

ちなみに名詞だと「倍数」らしいので、「エンジン倍数かどうか」という意味になります。

Reply:
なるほど、命名大事ですね